backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -125,8 +125,10 @@
           .append(ModelDBConstants.PROJECT_IDS)
           .append(")")
           .toString();
-  private static final String GET_PROJECT_BY_ID_HQL = "From ProjectEntity p where p.id = :id";
-  private static final String GET_PROJECT_BY_IDS_HQL = "From ProjectEntity p where p.id IN (:ids)";
+  private static final String GET_PROJECT_BY_ID_HQL =
+      "From ProjectEntity p where p.id = :id AND p.deleted = false";
+  private static final String GET_PROJECT_BY_IDS_HQL =
+      "From ProjectEntity p where p.id IN (:ids) AND p.deleted = false";
```

Comment:
Should we use ModelDBConstants.DELETED here too?

Reply:
updated with constant name